grafana/provisioning/dashboards/qumulo/node.json

```diff
@@ -1400,8 +1463,8 @@
       {
         "current": {
           "selected": false,
-          "text": "",
-          "value": ""
+          "text": "bw13:8000",
```

Comment:
There some data that made its way in here. I thought exporting for sharing wasn't supposed to include this.